.github/workflows/pr-ci.yml

```diff
@@ -1,85 +1,153 @@
 name: CI build and push (PR)
 
 concurrency:
-  group: pr-${{ github.event.number }}
+  group: ci-${{ github.run_id }}
   cancel-in-progress: true
 
 on:
-  pull_request_target:
+  issue_comment:
     types:
-      - opened
-      - reopened
-      - synchronize
-      - labeled
-      - unlabeled
-    branches:
-      - main
-      - v[0-9]+
-      - v[0-9]+.[0-9]+
-      - cryostat-v[0-9]+.[0-9]+
+      - created
 
 jobs:
+  check-before-build:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/build_test')
+    steps:
+    - name: Fail if needs-triage label applied 
+      if: ${{ contains(github.event.issue.labels.*.name, 'needs-triage') }}
+      run: exit 1
+    - name: Check command permission allowed
+      if: |
+        github.event.comment.author_association != 'MEMBER' &&
+        github.event.comment.author_association != 'OWNER' &&
+        !contains(github.event.issue.labels.*.name, 'safe-to-test')
+      uses: thollander/actions-comment-pull-request@v2
+      with:
+        message: |-
+          You do not have permission to run the /build_test command. Please ask Cryostat @reviewers 
+          to resolve the issue.
+    - name: Fail if command permission is not allowed
+      if: |
+        github.event.comment.author_association != 'MEMBER' &&
```

Comment:
We should also have a check so that only member, owner, and the author themselves can run the command in their own PR.

This prevents anyone from randomly run the command.

Reply:
Also, should it be `github.event.issue.comment.author_association` tho? 

https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#issue_comment